File: lab009/cliente/models/ingeniero.dart
class Ingeniero {
  final int? id;
  final String nombres;
  final String apellidos;
  final String especialidad;
  final String cargo;

  Ingeniero({this.id, required this.nombres, required this.apellidos, required this.especialidad, required this.cargo});

  factory Ingeniero.fromJson(Map<String, dynamic> json) {
    return Ingeniero(
      id: json['idIng'],
      nombres: json['nombres'] as String,
      apellidos: json['apellidos'] as String,
      especialidad: json['especialidad'] as String,
      cargo: json['cargo'] as String,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'idIng': id,
      'nombres': nombres,
      'apellidos': apellidos,
      'especialidad': especialidad,
      'cargo': cargo,
    };
  }
}

File: lab009/servidor/models/Ingeniero.java
package com.ocho.miguel.models;

public class Ingeniero {
    int idIng;
    String nombres;
    String apellidos;
    String especialidad;
    String cargo;

    public Ingeniero(int idIng, String nombres, String apellidos, String especialidad, String cargo) {
        this.idIng = idIng;
        this.nombres = nombres;
        this.apellidos = apellidos;
        this.especialidad = especialidad;
        this.cargo = cargo;
    }

    public int getIdIng() {
        return idIng;
    }

    public void setIdIng(int idIng) {
        this.idIng = idIng;
    }

    public String getNombres() {
        return nombres;
    }

    public void setNombres(String nombres) {
        this.nombres = nombres;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public String getEspecialidad() {
        return especialidad;
    }

    public void setEspecialidad(String especialidad) {
        this.especialidad = especialidad;
    }

    public String getCargo() {
        return cargo;
    }

    public void setCargo(String cargo) {
        this.cargo = cargo;
    }
}
